flutter/ios/Classes/SwiftSentryFlutterPlugin.swift

```diff
@@ -20,19 +29,31 @@ public class SwiftSentryFlutterPlugin: NSObject, FlutterPlugin {
     }
 
     private func registerObserver() {
+#if os(iOS)
         NotificationCenter.default.addObserver(self,
                                                selector: #selector(applicationDidBecomeActive),
                                                name: UIApplication.didBecomeActiveNotification,
                                                object: nil)
+#elseif os(macOS)
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(applicationDidBecomeActive),
+                                               name: NSApplication.didBecomeActiveNotification,
+                                               object: nil)
+#endif
```

Comment:
would be possible to this condition only once and save it to a variable? so we don't need to repeat it on registering/unregistering

Reply:
Like
```swift

#if os(iOS)
        var name = UIApplication.didBecomeActiveNotification;
#elseif os(macOS)
        var name = NSApplication.didBecomeActiveNotification;
#endif
        NotificationCenter.default.addObserver(self,
                                               selector: #selector(applicationDidBecomeActive),
                                               name: name,
                                               object: nil)

```
I think that should be doable

Reply:
yep, actually, we do that already on the Cocoa SDK